src/axom/inlet/Field.cpp

```diff
@@ -13,7 +14,7 @@ namespace axom
 {
 namespace inlet
 {
-std::shared_ptr<Field> Field::required(bool isRequired)
+std::shared_ptr<VerifiableScalar> Field::required(bool isRequired)
```

Comment:
Now that we have changed the `required()` to `isRequired()`, we can make this default to true